setup.py

```diff
@@ -102,11 +105,15 @@ def get_version(version_file):
             # 1.11.1 fixed this issue, but to ensure expected behavior we'll pin
             # to never grab the regression version.
             "coveralls != 1.11.0",
+            "docker == 7.1.0",
```

Comment:
Why does this one test dependency have a hard-coded version?  If a particular version is required then there should be a comment explaining why.

Reply:
Fixed. 
See:
- 8fff2b4a827029cb75dee35f4f4ebe3b1b47ad30

Reply:
See commit 8fff2b4.